components/Componenten/table/table.njk

```diff
@@ -1,56 +1,89 @@
-        GitHub gebruikersnaam 
-        {% if columnAlign === 'dso-text-left' %} (column left aligned)
-        {% elif columnAlign === 'dso-text-center' %} (column centered)
-        {% elif columnAlign === 'dso-text-right' %} (column right aligned)
-        {% endif %}
-        </th>
-      <th scope="col">Modifier</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <th scope="row">1</th>
-      <td><a href="#fabien">Fabien</a></td>
-      <td>Potencier</td>
-      <td {{ className(columnAlign) }}>fabpot</td>
-      <td>
-        <code>tr</code>
-      </td>
-    </tr>
-    <tr>
-      <th scope="row">2</th>
-      <td><a href="#andrew">Andrew</a></td>
-      <td>Nesbitt</td>
-      <td {{ className(columnAlign) }}>andrew</td>
-      <td>
-        <code>tr</code>
-      </td>
-    </tr>
-    <tr {{ className([activeRow, 'active']) }}>
-      <th scope="row">3</th>
-      <td><a href="#taylor">Taylor</a></td>
-      <td>Otwell</td>
-      <td {{ className(columnAlign) }}>taylorotwell</td>
-      <td>
-        <code>tr{% if activeRow %}.active{% endif %}</code>
-      </td>
-    </tr>
-    <tr>
-      <th scope="row">4</th>
-      <td><a href="#hugo">Hugo</a></td>
-      <td>Giraudel</td>
-      <td {{ className(columnAlign) }}>HugoGiraudel</td>
-      <td>
-        <code>tr</code>
-      </td>
-    </tr>
-  </tbody>
-</table>
+{%if filter %}
+  {% render '@search-bar', {icon: icon, placeholder: placeholder, value: value, button: button, small: small} %}
+  <table {{ className(modifier) }} role="grid" aria-readonly="true">
+    <caption {{ className([not showCaption, 'sr-only']) }}>{{ caption }}</caption>
+    <thead>
+      <tr role="row">
+        {% for columnHeader in columnHeaders %}
+          <th scope="col" {% if columnHeader.sorteerbaar %} role="columnheader" aria-sort="ascending"{% endif %}>
+            {{ columnHeader.title }}
+          </th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      {% if resultError %}
+        <tr class="result-error" role="row">
+          <td colspan="{{ cols }}">{{ resultError }}</td>
+        </tr>
+      {% elif rowContent %}
+        {% for content in rowContent %}
+          <tr {{ className(content.rowNotification) }} role="row">
+            {% if content.rowHeader %}
+              <th scope="row">{{ content.rowHeader }}</th>
+            {% endif %}
+            <td><a href="#">{{ content.bron }}</a></td>
+            <td>{{ content.uitleg }}</td>
+          </tr>
+        {% endfor %}
+      {% endif %}
+    </tbody>
+  </table>
+{%else%}
+  <table {{ className(modifier) }}>
+    <caption {{ className([not showCaption, 'sr-only']) }}>{{ caption }}</caption>
+    <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Voornaam</th>
+        <th scope="col">Achternaam</th>
+        <th scope="col" {{ className(columnAlign) }}>
+          GitHub gebruikersnaam 
+          {% if columnAlign === 'dso-text-left' %} (column left aligned)
```

Comment:
Leestekst doen we in de toolkit toch in het Nederlands?

Reply:
Daar twijfelde ik over, omdat het zo gelinkt is aan de markup van het voorbeeld. Ik zal het aanpassen.